src/components/calcite-label/calcite-label.tsx

```diff
@@ -30,6 +30,9 @@ export class CalciteLabel {
   //
   //--------------------------------------------------------------------------
 
+  /** specify the for(htmlFor) attribute (optional) */
```

Comment:
I removed `htmlFor` from `calcite-label` because I realized when I was writing accessibility tests, that the `<label>` element isn't actually able to be used alongside the hidden `<input type="radio">` because it's rendered within a shadow root.

I opted to use `aria-label` to get the accessibility tests to pass to overcome this limitation, so it turns out there's no need to add this attribute.